Automattic/jetpack
# Button block: Fix missing class issue. Closes #18325

Fixes #18325

#### Changes proposed in this Pull Request:
Fixes the issue with the button block missing the `wp-block-button` class on the frontend (it's present in the editor) - causing styling issues in themes.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
1. Activate the Twenty Twenty One theme
2. Create a page
3. Add a Mailchimp and/or Contact Form block
4. Check the button styling in the editor and on the frontend - notice the styling discrepancy
5. Apply the fix - notice that the button is now styled consistently

Before:
<img width="1542" alt="before" src="https://user-images.githubusercontent.com/905781/104517606-9d0b2080-55f6-11eb-9a9b-65ff505c2d8f.png">

After:
<img width="1510" alt="after" src="https://user-images.githubusercontent.com/905781/104517675-c330c080-55f6-11eb-8ce9-ac9d836d9b3c.png">

#### Proposed changelog entry for your changes:
No update in the changelog necessary
